### PR TITLE
Python SessionUT added

### PR DESCRIPTION
**Pull request by Fredooooooo**

### CODE REVIEW

**Review by jt2594838 — APPROVED** [review]
It is a good effort, but could our CI trigger it?

File: importerCSV-py/src/utils/Field.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
+from IoTDBConstants import TSDataType
+
+
+class Field(object):
```
Comment (HTHou) [inline importerCSV-py/src/utils/Field.py:4]: This file is missing Apache header, please take a look.

File: importerCSV-py/src/utils/IoTDBConstants.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
+from enum import Enum, unique
+
+
+@unique
```
Comment (HTHou) [inline importerCSV-py/src/utils/IoTDBConstants.py:5]: Same as above.

File: importerCSV-py/src/utils/IoTDBRpcDataSet.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
+import sys
+from IoTDBConstants import *
+sys.path.append("../../target")
+
+from thrift.transport import TSocket, TTransport
```
Comment (HTHou) [inline importerCSV-py/src/utils/IoTDBRpcDataSet.py:5]: Same..

File: importerCSV-py/src/utils/RowRecord.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+from IoTDBConstants import TSDataType
+from Field import Field
+
+
+class RowRecord(object):
+
```
Comment (HTHou) [inline importerCSV-py/src/utils/RowRecord.py:6]: Same

File: importerCSV-py/src/utils/Tablet.py [hunk 5]
```diff
@@ -0,0 +1,116 @@
+
+from IoTDBConstants import *
+import struct
+
+
```
Comment (HTHou) [inline importerCSV-py/src/utils/Tablet.py:5]: Same

**Review by HTHou — COMMENTED** [review]
Please check the missing Apache headers. And I don't think it's necessary to include binary files in project.

**Comment by jixuan1989** [comment]
this PR will be closed because of  #2539 

However, the PR e2e test is needed.

**Pull request closed without merging**